packages/AlgernonTests.package/ALGTest.class/instance/currentBrowsers.st

```diff
@@ -4,4 +4,4 @@ currentBrowsers
 	^ (ActiveWorld submorphs
 		select: [:each | (each isKindOf: SystemWindow)
 				and: [each model inheritsFromAnyIn: 
-					{Browser. PreferenceBrowser. MCWorkingCopyBrowser}]]) asSet
\ No newline at end of file
+					{Browser. PreferenceBrowser. MCWorkingCopyBrowser . TranscriptStream . MessageNames}]]) asSet
```

Comment:
Could you refactor this code to just close all newly opened `PluggableSystemWindow`s? That should do it without us having to manually specify each tool.

Reply:
Sure, I can do that.

Reply:
I just committed the new "currentSystemWindows"-Method.